flash/pointcloud/__init__.py

```diff
@@ -0,0 +1,6 @@
+from flash.pointcloud.segmentation.data import PointCloudSegmentationData  # noqa: F401
+from flash.pointcloud.segmentation.data import PointCloudSegmentationDatasetDataSource  # noqa: F401
+from flash.pointcloud.segmentation.data import PointCloudSegmentationFoldersDataSource  # noqa: F401
+from flash.pointcloud.segmentation.data import PointCloudSegmentationPreprocess  # noqa: F401
```

Comment:
We normally just do the model and the datamodule right?
```suggestion
```

Reply:
Done !